commands/bootstrap/inputs.go

```diff
@@ -596,7 +613,47 @@ func initializeInputs(inputFilePath, resourcesPath string, configuration config.
 		}
 	}
 
+	if inputsPath == "" {
+
+		bSlice, err := yaml.Marshal(inputValues)
+
+		inputsPathOut := deploymentName + ".yaml"
+
+		count := 1
+		for {
+			if _, err := os.Stat(inputsPathOut); os.IsNotExist(err) {
+				break
+			}
+			inputsPathOut = deploymentName + "_" + strconv.Itoa(count) + ".yaml"
+			count++
+		}
+
+		if _, err := os.Stat("inputsaves/"); os.IsNotExist(err) {
+			err = os.Mkdir("inputsaves/", 0755)
+			if err != nil {
+				log.Fatal("cannot create inputsave directory.", err)
+			}
+		}
+
+		file, err := os.Create("inputsaves/" + inputsPathOut)
+
+		println("Exporting set configuration to inputsaves/" + inputsPathOut)
```

Comment:
It is generally better to have the error check just after getting the error.
Will prevent issues in refactoring.

Reply:
Done, I moved the println("exporting ... right before os.create